hail/python/hailtop/utils/utils.py

```diff
@@ -107,7 +107,7 @@ def digits_needed(i: int) -> int:
     return 1 + digits_needed(i // 10)
 
 
-def grouped(n, ls):
+def grouped(n:int, ls:list[Any]) -> list:
```

Comment:
A couple of things

1. Type annotations have an int after the colon
2. Types are all lowercase for things like int, str, float, bytes, but are in upper case (imported from Typing above) for things like Any, List, Dict, Tuple, Iterable, etc.
3. Think about what `grouped` is returning. Take a look at your test cases you wrote a week or two ago.